src/frontend/src/components/landingPage.ts

```diff
@@ -0,0 +1,58 @@
+import { I18n } from "$src/i18n";
+import { html, TemplateResult } from "lit-html";
+import { navigationLink } from "./footer";
+import { githubIcon, icLogo, questionIcon } from "./icons";
+import copyJson from "./landingPage.json";
+
+/**
+ * Landing page template
+ *
+ * It is a component with split panes left and right in desktop view.
+ * To the left there is some static content, and to the right there is a slot.
+ *
+ * In movile view, the static content is below the slot.
+ *
+ */
+export const landingPage = ({
+  slot,
+}: {
+  slot: TemplateResult;
+}): TemplateResult => {
+  const i18n = new I18n();
+  const copy = i18n.i18n(copyJson);
+
+  return html` <main class="c-landingPage">
+    <div class="c-landingPage__logo">
+      <div class="c-logo">${icLogo}</div>
+    </div>
+    <section class="c-landingPage__right">
+      <div class="c-landingPage__right__content">
+        <div class="c-landingPage__right__content--full-width">${slot}</div>
+      </div>
+    </section>
+    <section class="c-landingPage__left">
+      <div class="c-landingPage__left__content">
+        <h1 class="t-title t-title--main">${copy.title}</h1>
+        <p class="t-paragraph">{${copy.subtitle}}</p>
+      </div>
+      <div class="c-landingPage__left__footer">
```

Comment:
this the main I think not the thing around...

Reply:
what do you mean?

Reply:
main is only supposed to be the actual main content of the page. Not everything including the logo and this marketing stuff on the left